apis/Google.Cloud.Logging.Log4Net/Google.Cloud.Logging.Log4Net.Tests/Log4NetTest.cs

```diff
@@ -29,24 +29,16 @@
 using System.Runtime.CompilerServices;
 using System.Threading;
 using System.Threading.Tasks;
-using Xunit;
-
+using Xunit;
+
```

Comment:
This diff is almost certainly just a CRLF vs LF difference. The file was inconsistent before. It's now all-Unix. We may want to change it to all-Windows at some point...